consoles/node.html

```diff
@@ -10,7 +10,25 @@
 {{template "prom_content_head" .}}
 <h1>Node</h1>
 
-Choose an instance on the left.
+<table class="table table-condensed table-striped" style="width: 0%">
```

Comment:
I think with an added class of "table-bordered", the table looks better in the context in which it is displayed.